doc/architecture/protocol.md

```diff
@@ -298,3 +298,4 @@ These also accept `POST`. The below endpoints all use `GET`:
 * `/fuzzydelhash` - Deletes entries from fuzzy according to content of `Hash:` header(s)
 * `/plugins` - Returns list of plugins or plugin specific stuff
 * `/ping` - Returns just a `pong` HTTP reply (could be used for monitoring)
+* `/metrics` - Returns OpenMetrics data
```

Comment:
One neat here: maybe we should describe all counters/gauges etc that are returned by Rspamd within this endpoint? A simple copy&paste from the output comments would be enough I suppose.